src/main/java/fr/ospiea/oscra/employee/object/Employee.java

```diff
@@ -0,0 +1,113 @@
+// tag::sample[]
+package fr.ospiea.oscra.employee.object;
+
+import fr.ospiea.oscra.common.AbstractEntity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+public class Employee extends AbstractEntity implements Serializable{
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+    @Enumerated(EnumType.STRING)
+    private EnabledType enabled;
+
+    @Override
+    public String toString() {
+        return "Employee{" +
+                "firstName='" + firstName + '\'' +
+                ", lastName='" + lastName + '\'' +
+                ", email='" + email + '\'' +
+                ", password='" + password + '\'' +
+                ", enabled=" + enabled +
+                ", role=" + role +
+                ", address=" + address +
+                ", employeeDetail=" + employeeDetail +
+                '}';
+    }
+
+    public EnabledType getEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(EnabledType enabled) {
+        this.enabled = enabled;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    @Enumerated(EnumType.STRING)
+    private Role role;
+
+    @Embedded
+    private Address address;
+
+    public Address getAddress() {
+        return address;
+    }
+
+    @Embedded
+    private EmployeeDetail employeeDetail;
```

Comment:
For me, if we put address, employeedetail in employee, the class employee has too many lines.